drivers/hmis/app/models/hmis/role.rb

```diff
@@ -380,8 +380,15 @@ def self.permissions_with_descriptions
         category: 'Administration',
         sub_category: 'Enrollment Management',
       },
+      can_manage_forms: {
+        description: 'Ability to edit forms',
+        administrative: true,
+        access: [:editable],
+        category: 'Administration',
+        sub_category: 'Admin Tools',
+      },
       can_configure_data_collection: {
-        description: 'Ability to configure custom assessments, services, auto-exit, and other application rules',
+        description: 'Ability to configure rules for custom assessments, services, auto-exit, etc, but not to edit forms',
```

Comment:
The "etc" feels a little too informal. I think just the "rules" change is probably sufficient.
```suggestion
        description: 'Ability to configure data collection rules for assessments, services, auto-exit, and more.',
```
maybe? We will probably want to re-work the description once we've reached a "v1" state for the config tool.